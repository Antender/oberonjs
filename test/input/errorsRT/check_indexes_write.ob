(*options:{"checkIndexes": true}*)
MODULE m;
VAR
	s: ARRAY 1 OF CHAR;

PROCEDURE putCharAt(VAR a: ARRAY OF CHAR; i: INTEGER);
BEGIN
	a[i] := "a";
END putCharAt;

BEGIN
	putCharAt(s, 1);
END m.