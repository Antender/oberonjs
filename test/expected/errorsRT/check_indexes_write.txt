Error: index out of bounds: 1